packages/discord.js/src/structures/Message.js

```diff
@@ -681,31 +681,33 @@ class Message extends Base {
 
   /**
    * Pins this message to the channel's pinned messages.
+   * @param {string} [reason] Reason for pinning
    * @returns {Promise<Message>}
    * @example
    * // Pin a message
    * message.pin()
```

Comment:
I would add a reason to the example, same below

Reply:
If that's to be done, it should be a separate example, otherwise people will understand it's required when it's optional.